salalib/mgraph.cpp

```diff
@@ -1278,7 +1278,7 @@ bool MetaGraph::analyseAxial( Communicator *communicator, Options options, bool
    bool retvar = false;

```

Comment:
I am assuming `retvar` already gets used further down the line? Could we also rename it to something more meaningful while in there, e.g. `hasOutput`?